common/src/main/scala/com/gu/media/model/PlutoIntegrationData.scala

```diff
@@ -26,7 +28,14 @@ object AtomAssignedProjectMessage {
 
   def build(atom: MediaAtom): AtomAssignedProjectMessage = {
     val plutoData = atom.plutoData.get
-    AtomAssignedProjectMessage("project-assigned", atom.id, plutoData.commissionId.get, plutoData.projectId.get)
+
+    val email = for {
```

Comment:
Do we really need a for comprehension here? `atom.contentChangeDetails.created.flatMap(_.user.map(_.email))` should be ok?

Reply:
i was quite pleased with that, normally i can't make the damn things work 😄 thought it would be more readable than the flatmap/map